packages/icons/src/Account.svg

```diff
@@ -0,0 +1 @@
+<svg fill="currentColor" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><path fill-rule="evenodd" clip-rule="evenodd" d="M8.18 1.1a.485.485 0 0 0-.5-.002l-6.044 3.61a.501.501 0 0 0-.001.86l1.668 1.005-1.667.995a.501.501 0 0 0-.001.86l1.723 1.039-1.722 1.029a.501.501 0 0 0-.001.86l5.981 3.603a.485.485 0 0 0 .5.001l6.035-3.603a.502.502 0 0 0 .002-.86l-1.722-1.04 1.72-1.027a.501.501 0 0 0 .002-.86l-1.666-1.007 1.664-.994a.501.501 0 0 0 .002-.86L8.18 1.099Zm3.283 8.934-3.347 1.999a.486.486 0 0 1-.5-.001l-3.291-1.983-1.472.88 5.015 3.02 5.067-3.025-1.472-.89ZM7.868 8.161l5.067-3.025-5.008-3.027L2.853 5.14l5.015 3.021Z" fill="#4F5162"/></svg>
```

Comment:
the fill should be set to currentColor on the path.. not on the entire svg element.
we should remove the hex color codes in the last line